docs/user/installation_and_configuration.rst

```diff
@@ -107,8 +107,12 @@ Installing the Accumulo Distributed Runtime Library
 The ``geomesa-accumulo-dist_2.11-$VERSION/dist/accumulo/`` directory contains the distributed
 runtime JARs that contains server-side code for Accumulo that must be made
 available on each of the Accumulo tablet servers in the cluster. These JARs
-contain GeoMesa code and the Accumulo iterator required for querying
-GeoMesa data.
+contain GeoMesa code and the Accumulo iterators required for querying GeoMesa data.
+
+.. warning::
+
+    GeoMesa requires commons-vfs2.jar 2.1 or later. This jar ships with Accumulo 1.7.2+, but for older
```

Comment:
[picky style] I've been trying to use "JAR" rather than "jar" whenever possible.